src/components/organisms/TimingDeleteModal/TimingDeleteModal.tsx

```diff
@@ -71,42 +71,37 @@ export const TimingDeleteModal: React.FC<TimingDeleteModalProps> = ({
       )
     : "Unknown";
   const eventDuration = event
-    ? `${event.durationMinutes / 60} hours ${
+    ? `${(event.durationMinutes / 60).toFixed()} hours ${
         event.durationMinutes % 60
       } minutes`
     : "Unknown";
```

Comment:
[OOS] There are probably functions in library (maybe even libraries included in this project) that turn date objects into human readable "... hours ago" type of messages